mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -691,6 +691,7 @@ static ParseResult parseMapClause(OpAsmParser &parser, IntegerAttr &mapType) {
     if (parser.parseKeyword(&mapTypeMod))
       return failure();
 
+    // Map-type-modifiers
```

Comment:
I appreciate the comment but it can be a bit misleading, `implicit` isn't necessarily a map type modifier.